test/trackler/problems_test.rb

```diff
@@ -14,13 +14,13 @@ def test_collection
 
     # kind of dirty set operation
     slugs = %w(one two three apple)
-    todos = %w(dog hello-world imbe banana cherry mango).sort
+    todos = %w(dog hello-world imbe banana cherry mango track-and-field).sort
     assert_equal todos, problems - slugs
   end
 
   def test_only_valid_problems
     problems = Trackler::Problems.new(FIXTURE_PATH)
-    slugs = %w(apple banana cherry dog hello-world imbe mango one three two)
+    slugs = %w(apple banana cherry dog hello-world imbe mango one three track-and-field two)
```

Comment:
By not adding the track-and-field problem, you avoid having to change `problems_test` and `trackler_test`